lib/capybara/poltergeist/inspector.rb

```diff
@@ -22,9 +22,12 @@ def url
       "//localhost:#{port}/"
     end
 
-    def open
+    def open(scheme = nil)
       if browser
-        Process.spawn(browser, url)
+        scheme = 'http' unless scheme == 'https' # scheme could be 'file' or 'about' or something else
```

Comment:
if scheme can only be http or https why allow the user to pass in anything they want to the method?